packages/fast-components-class-name-contracts-base/src/viewport-positioner/index.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * The class name contract for the viewport positioner component
+ */
+export interface ViewportPositionerClassNameContract {
+    /**
+     * The root of the viewport positioner component
+     */
+    viewportPositioner?: string;
+
+    /**
+     * left positioner horizontal state
+     */
+    viewportPositioner__left?: string;
+
+    /**
+     * right positioner horizontal state
+     */
+    viewportPositioner__right?: string;
+
+    /**
+     * top positioner vertical state
+     */
+    viewportPositioner__top?: string;
+
+    /**
+     * bottom positioner vertical state
+     */
+    viewportPositioner__bottom?: string;
+
+    /**
+     * horizontal flip outward state
+     */
+    viewportPositioner__horizontalFlipOutward?: string;
+
+    /**
+     * horizontal flip inward state
+     */
+    viewportPositioner__horizontalFlipInward?: string;
+
+    /**
+     * vertical flip outward state
+     */
+    viewportPositioner__verticalFlipOutward?: string;
```

Comment:
I'm thinking maybe we should follow the CSS naming convention, where "flip outward" is the default and you only specify flip inward, which could be renamed to "inset"

Reply:
Yep, "inset" is much better